pipeline/customer/customer.py

```diff
@@ -95,6 +101,13 @@ def run(argv=None):
         type=str,
         help='Output file pattern',
         required=True)
+    parser.add_value_provider_argument(
+        '--output_aggregation',
+        type=str,
+        help='Aggregation that is outputted',
+        choices=['min', 'max', 'sum', 'avg'],
+        default='sum'
```

Comment:
That is one way to do it. Another way would be to not set a default value but mark it as required. This way, we could show that this required parameter need not be provided during template compilation.

Reply:
Removed the `default`. But not added the required because otherwise we have to enter the value during the compilation. But in the metadata file it is required.

Reply:
Hmm, ok. I remembered it differently but you might be right.